content/fr/docs/intro/install.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "Installation de Helm"
+description: "Learn how to install and get running with Helm."
+weight: 2
+---
+
+Ce guide vous explique comment installer la CLI (Interface de Ligne de Commande) Helm. Helm peut être installé soit à partir des sources, ou à partir des releases binaires pré-construites.
+
+## Depuis le projet Helm
+
+Nous vous proposons deux méthodes pour récupérer et installer Helm. Voici les méthodes officielles pour obtenir les releases de l'application. En plus de cela, la communauté Helm
+fournit des méthodes pour installer Helm via différents gestionnaires de packages. L'installation via ces méthodes peut être trouvée ci-dessous.
+
+### À partir des releases binaires
+
+Chaque [releases] (https://github.com/helm/helm/releases) de Helm fournit des binaires pour une variété de systèmes d'exploitation. Ces binaires peuvent être téléchargées manuellement
+et installé.
+
+1. Téléchargez la [version adéquate](https://github.com/helm/helm/releases)
+2. Décompressez l'archive (`tar -zxvf helm-v3.0.0-linux-amd64.tar.gz`)
+3. Trouvez le binaire `helm` dans le répertoire décompréssé, et déplacez le dans la destination voulue (`mv linux-amd64/helm /usr/local/bin/helm`)
+
+A ce stade, vous devriez être cappable de lancer le client et [d'ajouter le dépot stable](https://helm.sh/docs/intro/quickstart/#initialize-a-helm-chart-repository): `helm help`.
+
+**Remarque:** Les tests automatisés de Helm sont effectués pour Linux AMD64 uniquement pendant les build et release CircleCi. Les tests d'autres systèmes d'exploitation sont sous la responsabilité de la communauté qui demande Helm pour le système d'exploitation en question.
+
+### À partir du script
+
+Helm a maintenant un script d'installation qui récupérera automatiquement la dernière version de Helm et [l'installera localement](https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3).
+
+Vous pouvez récupérer ce script, puis l'exécuter localement. Il est bien documenté donc que vous pouvez le lire et comprendre ce qu'il fait avant de l'exécuter.
+
+```console
+$ curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
+$ chmod 700 get_helm.sh
+$ ./get_helm.sh
+```
+
+Oui, vous pouvez également `curl https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash` si vous aimez vivre dangereusement.
+
+## Via les gestionnaires de packages
+
+La communauté Helm offre la possibilité d'installer Helm via le gestionnaires de packages de votre système d'exploitation. Ceux-ci ne sont pas pris en charge par le projet Helm et ne sont pas considérés comme des tiers de confiance.
+
+### Depuis Homebrew (macOS)
+
+Les membres de la communauté Helm ont contribué à la création d'une formule Helm pour Homebrew. Ce package est généralement à jour.
+
+```console
+brew install helm
+```
+
+(Note: There is also a formula for emacs-helm, which is a different project.)
```

Comment:
Did you want to keep this note as English?

Reply:
it is an oversight on my part !